README.md

```diff
@@ -1,18 +1,1110 @@
+import static com.kms.katalon.core.testcase.TestCaseFactory.findTestCase
+import static com.kms.katalon.core.checkpoint.CheckpointFactory.findCheckpoint
+import com.kms.katalon.core.model.FailureHandling as FailureHandling
+import com.kms.katalon.core.testcase.TestCase as TestCase
+import com.kms.katalon.core.testdata.TestData as TestData
+import com.kms.katalon.core.testobject.TestObject as TestObject
+import com.kms.katalon.core.checkpoint.Checkpoint as Checkpoint
+import internal.GlobalVariable as GlobalVariable
+import com.kms.katalon.core.configuration.RunConfiguration as RunConfiguration
+
+// To specify the path to data files
+String path = RunConfiguration.getProjectDir() + '/Data Files/'
+
+```
+
+### Use PDF Keyword Plugin
+
+PDF keywords Plugin offers 14 custom keywords to help you start automating the test with PDF documents. 
+
+- To get the total number of pages of the PDF documents, use the following keyword:
+
+<details><summary> The <code>getPageNumber</code> keyword </summary>
+
+### Description
+
+This keyword gets the total number of pages in the document.
+### Parameters
+
+<table width="854">
+<tbody>
+<tr>
+<td><strong>Parameter</strong></td>
+<td><strong>Type</strong></td>
+<td><strong>Mandatory</strong></td>
+<td><strong>Description</strong></td>
+</tr>
+<tr>
+<td>File</td>
+<td>
+<p>String</p>
+</td>
+<td>Required</td>
+<td>The absolute file path</td>
+</tr>
+</tbody>
+</table>
+
+### Example
+
+To get the total number of pages of the **Test_text1.pdf** PDF document, copy and paste the sample code as below:
+
+``` groovy
+import com.kms.katalon.core.util.KeywordUtil as KeywordUtil
+import com.kms.katalon.core.webui.keyword.WebUiBuiltInKeywords as WebUI
+import com.kms.katalon.core.mobile.keyword.MobileBuiltInKeywords as Mobile
+import com.kms.katalon.core.cucumber.keyword.CucumberBuiltinKeywords as CucumberKW
+import com.kms.katalon.core.webservice.keyword.WSBuiltInKeywords as WS
+import static com.kms.katalon.core.testobject.ObjectRepository.findTestObject
+import static com.kms.katalon.core.testdata.TestDataFactory.findTestData
+import static com.kms.katalon.core.testcase.TestCaseFactory.findTestCase
+import static com.kms.katalon.core.checkpoint.CheckpointFactory.findCheckpoint
+import com.kms.katalon.core.model.FailureHandling as FailureHandling
+import com.kms.katalon.core.testcase.TestCase as TestCase
+import com.kms.katalon.core.testdata.TestData as TestData
+import com.kms.katalon.core.testobject.TestObject as TestObject
+import com.kms.katalon.core.checkpoint.Checkpoint as Checkpoint
+import internal.GlobalVariable as GlobalVariable
+import com.kms.katalon.core.configuration.RunConfiguration as RunConfiguration
+
+String path = RunConfiguration.getProjectDir() + '/Data Files/'
+
+CustomKeywords.'com.kms.katalon.keyword.pdf.PDF.getPageNumber'(path + 'test_text1.pdf')
+
+```
+</details>
+&nbsp;
+
+- To get the content of the PDF file as plain text, use the following keywords:
```

Comment:
To format the content of your PDF files as plain text, use any of the following keywords: